pkg/vpcmodel/jsonOutput.go

```diff
@@ -108,3 +122,85 @@ func getConnLinesForSubnetsConnectivity(conn *VPCsubnetConnectivity) []connLine
 	sortConnLines(connLines)
 	return connLines
 }
+
+type allSemanticDiff struct {
+	SemanticDiff []diffLine `json:"semantic_diff"`
+}
+
+func getDiffLines(configsDiff *diffBetweenCfgs) []diffLine {
+	diffLines := getDirectionalDiffLines(configsDiff.cfg1ConnRemovedFrom2, true)
+	diffLines = append(diffLines, getDirectionalDiffLines(configsDiff.cfg2ConnRemovedFrom1, false)...)
+	return diffLines
+}
+
+func getDirectionalDiffLines(connectDiff connectivityDiff, thisMinusTheOther bool) []diffLine {
```

Comment:
after the merge from `main` branch -- `thisMinusTheOther` is part of `connectionDiff` object, so there is no need to send it as an argument to this function.

Reply:
right